### PR TITLE
[WP#48988]Write Document for getting file information from file id

### PR DESCRIPTION
**Pull request by SagarGi**
### Description
This PR adds the documentation regarding getting the information of the file with `fileid(s)`.

### Related WorkPackage
https://community.openproject.org/work_packages/48988/activity

### CODE REVIEW

**Comment by SagarGi** [comment]
Currently CI is failing and the fix is in this PR https://github.com/nextcloud/groupfolders/pull/2542

File: README.md [hunk 1]
```diff
@@ -140,6 +140,108 @@ bash integration_setup.sh
 
 > Note: these credentials are only used by the script to do the setup. They are not stored/remembered.
 
+## Get file information
+We have two endpoints `fileinfo` and `filesinfo` from which we can get the information of a single or multiple files.
```
Comment (SwikritiT) [inline README.md:144]: Can we add a requirement section that mentions the Nextcloud activity app? I can see the notes but I think we expect the users to have activity app and tests also need activity app

**Comment by SwikritiT** [comment]
> Currently CI is failing and the fix is in this PR [nextcloud/groupfolders#2542](https://github.com/nextcloud/groupfolders/pull/2542)

@SagarGi https://github.com/nextcloud/integration_openproject/pull/490 this PR got merged so currently CI is not running in nextcloud master so rebase and push

File: README.md [hunk 2]
```diff
@@ -140,6 +140,107 @@ bash integration_setup.sh
...
+		  "modifier_name": <modifier_name>,
+		  "modifier_id": <modifier_id>,
+		  "dav_permissions": <dav_permissions>,
+		  "path": <path>
+		}
+	  }
+	}
+	```
+2. **Get information of multiple files**:
+Send the `POST` request to `filesinfo` endpoint with data `fileIds` of the files to retrieve information.
```
Comment (phil-davis) [inline README.md:186]: I suppose that the Activity app is also required for this request to work?

If so, then we need a general note about the Activity app - like @SwikritiT has mentioned in her comment above.

File: README.md [hunk 3]
```diff
@@ -140,6 +140,107 @@ bash integration_setup.sh
 
 > Note: these credentials are only used by the script to do the setup. They are not stored/remembered.
 
+## Get file information
+We have two endpoints `fileinfo` and `filesinfo` from which we can get the information of a single or multiple files.
+
```
Comment (SwikritiT) [inline README.md:145]: @SagarGi how about this and remove all the notes
```suggestion
### Requirements
This endpoint has a soft dependency on Nextcloud's Activity(link/to/activity/app) app. In the absence of the Activity app the response fields like `modifier_name` , `modifier_id` will be returned as null. So it is encouraged to have Activity app installed and enabled.


```

**Review by individual-it — APPROVED** [review]

**Review by SwikritiT — APPROVED** [review]

**Pull request merged**